app/views/teams/_form.html.erb

```diff
@@ -6,6 +6,7 @@
     <%= f.input :name %>
     <%= f.input :color, as: :fancy_color %>
     <%= f.association :team_type %>
+    <%= f.input :join_permission, collection: Team.join_permissions.keys %>
```

Comment:
Is there a way we can display a slightly friendlier looking "Added by Admins" instead of "added_by_admin" here?